utils/build.ps1

```diff
@@ -2334,6 +2349,8 @@ function Build-CURL([Hashtable] $Platform) {
       USE_WIN32_LDAP = "NO";
       ZLIB_ROOT = "$BinaryCache\$($Platform.Triple)\usr";
       ZLIB_LIBRARY = "$BinaryCache\$($Platform.Triple)\usr\lib\zlibstatic.lib";
+      BROTLIDEC_LIBRARY = "$BinaryCache\$($Platform.Triple)\usr\lib\brotlidec.lib"
+      BROTLICOMMON_LIBRARY = "$BinaryCache\$($Platform.Triple)\usr\lib\brotlicommon.lib"
```

Comment:
Do we need to pass in the library paths or do they generate a config file that we can use with `Brotli_DIR`?

Reply:
The build output of brotli does include 3 .pc files, but unsure how well curl supports BROTLI_DIR. The input variables for curl's brotli cmake are `BROTLI_INCLUDE_DIR`, `BROTLICOMMON_LIBRARY`, `BROTLIDEC_LIBRARY` according to https://github.com/curl/curl/blob/master/CMake/FindBrotli.cmake

Reply:
fixed in ab40c18 . I'm not sure why this works as I don't find a BrotliConfig.cmake in brotli's output. According to https://cmake.org/cmake/help/latest/command/find_package.html#full-signature , for this `<package>_DIR` pattern to work, BrotliConfig.cmake needs to be found in that directory.